doc/interactors.md

```diff
@@ -257,8 +258,8 @@ The IconButton element
 
 ##### Synopsis
 
-``` javascript
-import { IconButton } from '@folio/stripes-testing';
+```javascript
+import { IconButton } from "@folio/stripes-testing";
```

Comment:
What is the convention here in FOLIO land? we should stick with that.

Reply:
Single quotes appears to be the way.